src/components/presentational/Action/Action.tsx

```diff
@@ -45,7 +46,10 @@ export default function (props: ActionProps) {
 						{props.indicatorValue}&nbsp;
 					</span>
 				}
-				<FontAwesomeSvgIcon icon={indicatorIcon} />
+				{props.indicator}
+				{!props.indicator &&
```

Comment:
Should this be: `!props.indicator && props.indicatorIcon &&`

Reply:
No because there's a default indicator icon (the chevron); it is just that if you want an element and not a font awesome icon to by your indicator you can now just supply "indicator" (vs. indicatorIcon)

Reply:
Got it.  I missed the defaulting.